scaleapi/__init__.py

```diff
@@ -777,6 +785,9 @@ def get_batches(
                 "offset": offset,
```

Comment:
Since you defined a default value to `exclude_archived`  (`False`) during method initialization, we can add this to the `batches_args`?

```suggestion
                "offset": offset,
                "exclude_archived": exclude_archived,
```